src/desktop/apps/auction_support/client/registration_form.coffee

```diff
@@ -121,10 +121,12 @@ module.exports = class RegistrationForm extends ErrorHandlingForm
   validateAcceptCOS: (e) ->
     if @$acceptCOS.prop('checked')
       @$('.artsy-checkbox').removeClass('error')
+      @$('#cos_link').removeClass('conceal-hover-state')
```

Comment:
Is there any way to achieve the hover behavior without needing jquery?

Reply:
Updated to use z-index.